scripts/tools/tools-object.js

```diff
@@ -16,10 +16,10 @@ const options = {
 
 const fuse = new Fuse(categoryList, options)
 
-const createToolObject = (toolFile, repositoryUrl, isAsyncAPIrepo) => {
+const createToolObject = (toolFile, repositoryUrl, repoDescription, isAsyncAPIrepo) => {
   let resultantObject = {
     title: toolFile.title,
-    description: toolFile.description,
+    description: repoDescription || toolFile.description,
```

Comment:
My query is what if there are certain repos whose descriptions are not precise enough to show them to readers? There can be the case that the owner of the tool wants to make a custom description for its tool to show up in the dashboard. Then this will create problems.

Reply:
What problems? If someone will want to precise description will have to write description in the config file. I don't see any problems 😅

Reply:
Even if someone writes the description, it won't be considered with this implementation as it will only take the description mentioned in the GitHub repository.

Reply:
Ahh yes, It should be:

```
toolFile.description || repoDescription,
```

Reply:
good catch!!!